i-VRESSE/workflow-builder
# Apply toml schemas when creating an archive

While creating the archive the toml schemas from the catalog where not be passed. This PR passes them so the toml file is generated correctly.

To test
1. Add flexref node (any node with parameter which is an array should work)
2. Expand `molecule` 
3. In `mol_shape` add several items by pressing `+` button and check the checkbox them 
4. Download archive
5. In middle column switch from visual to text mode
6. The `workflow.cfg` inside the archive should be same as text shown in text mode.

Refs #65
